addons/mail/views/res_partner_views.xml

```diff
@@ -79,10 +79,10 @@
                     <field name="id"/>
                     <templates>
                         <div t-name="activity-box">
-                            <img t-att-src="activity_image('res.partner', 'avatar_128', record.id.raw_value)" role="img" t-att-title="record.id.value" t-att-alt="record.id.value"/>
-                            <div>
-                                <field name="name" display="full"/>
-                                <field name="parent_id" muted="1" display="full"/>
+                            <img class="rounded-circle" t-att-src="activity_image('res.partner', 'avatar_128', record.id.raw_value)" role="img" t-att-title="record.name.value" t-att-alt="record.name.value"/>
+                            <div class="ms-2">
+                                <field name="name" display="full" class="d-block"/>
+                                <field name="parent_id" muted="1" display="full" class="small"/>
```

Comment:
comment here but it applies to (all/most?) of them:

when there is a single line, it used to be centered vertically on the image, now it occupies the top position